### DISCUSSION TITLE
Cómo es el comportamiento de un survey/survey según en la posición que se encuentra éste?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#798] [2. Questions and Answers] [answered]
Hola a todos

En relación a la componente survey/survey,  nos percatamos lo siguiente:

1.- Si es un formulario, ésta se encuentra en el inicio (como componente primaria). Se visualiza y se registrada normalmente, pero cuando envías el formulario ésta quedará "vacía", como si no se registro el survey survey.

2.- Si en un formulario, ésta se encuentra entre componente secundaria, medio o final del formulario y se completa, no habrá problemas.

Adjuntamos un link de video que muestra ambos comportamiento:

link: https://drive.google.com/file/d/1TY3Ljp2hfbyW143m416UnTjfSzGYdJAn/view?usp=sharing

En la documentación de survey componente
link: https://doc.cotalker.com/docs/documentation/admin/survey/components/survey

No encontramos warning o algunas consideraciones a la hora de usarlo, por lo que no se sabe si:

1.- Es el comportamiento esperado, cuando se encuentra como componente principal y que falta sólo actualizar la documentación agregando estas consideraciones.

2.- Es un bug, ya que no funciona como componente principal.

Gracias =>

**Comment by acedward** [comment, marked as answer]
Bug reportado: https://github.com/Cotalker/documentation/issues/810
